dietpi/dietpi-software

```diff
@@ -1199,6 +1199,14 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_CATX[$software_id]=8
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/cloud/#k3s'
 
+		#------------------
+		software_id=200
+
+		aSOFTWARE_NAME[$software_id]='DietPi Dashboard (beta)'
```

Comment:
Another question: Shall we call it "DietPi-Dashboard" with a dash before "Dash", to align it with other script names, like "DietPi-CloudShell" and "DietPi-VPN"?

Reply:
Yeah, that will probably match things up better.